src/pages/Login.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, FormEvent, FC } from "react";
+import { Link } from "react-router-dom";
+import TextInput from "../components/TextInput.component";
+import PasswordInput from "../components/PasswordInput.component";
+import { auth } from "../utils/firebase/firebase.utils";
+import { useNavigate } from "react-router-dom";
+import { signInWithEmailAndPassword } from "firebase/auth";
+
+interface LoginProps {
+  setIsAuth: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const Login: FC<LoginProps> = ({ setIsAuth }) => {
+  const navigate = useNavigate();
+  const [values, setValues] = useState({
+    email: "",
+    password: "",
+  });
+  const [errorMsg, setErrorMsg] = useState("");
+  const [submitButtonDisabled, setSubmitButtonDisabled] = useState(false);
+  const handleChangeEmail = (e: FormEvent) => {
```

Comment:
An empty line to separate your state variables and functions helps give the code a cleaner look